src/daemon/rpmostreed-transaction-types.cxx

```diff
@@ -2707,19 +2710,20 @@ kernel_arg_apply_patching (KernelArgTransaction *self, RpmOstreeSysrootUpgrader
   for (char **iter = append_if_missing; iter && *iter; iter++)
     {
       const char *arg = *iter;
-      if (!ostree_kernel_args_contains (kargs, arg))
```

Comment:
I think this would be cleaner with an `ostree_kernel_args_contains_value` which doesn't strip off the value.